cve_bin_tool/cli.py

```diff
@@ -178,7 +201,12 @@ def main(argv=None):
                           """
         LOGGER.warning(warning_nolinux)
 
-    if not os.path.isfile(args.directory) and not os.path.isdir(args.directory):
+    if not args.directory and not args.input_file:
+        LOGGER.error("Error: Either a directory to scan or an input file required")
+        parser.print_usage()
+        return 0
+
+    if args.directory and not os.path.exists(args.directory):
         LOGGER.error("Error: directory/file invalid")
         parser.print_usage()
         return -1
```

Comment:
Now that we have error codes, we should use them here too.

Reply:
This is automatically getting returned as error codes because we are using sys.exit(main()) to run main function of cli.py

Reply:
We can use custom exception with with my new error code data structure if we also want to provide traceback here.